<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="framefondo">
      <property name="styleSheet">
       <string notr="true">QWidget#framefondo{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.039548 rgba(0, 67, 109, 255), stop:1 rgba(203, 203, 203, 255));}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_superior">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>911</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 85, 127);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>331</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Arial&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Gestión de inquilinos</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btnRegistrar">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>310</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 87 9pt &quot;Arial&quot;;
background-color: rgb(0, 68, 100);
color:white;

border-radius: 10px;
}


QPushButton:hover {
background-color:rgb(0, 101, 148);
}</string>
       </property>
       <property name="text">
        <string>Registrar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>./Imagenes/image-removebg-preview.png</normaloff>./Imagenes/image-removebg-preview.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>180</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>150</y>
         <width>221</width>
         <height>191</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>./Imagenes/inicio.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbInquilino">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>170</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Inquilino</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rbPropietario">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>170</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Propietario</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>200</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingrese su cédula:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtCedula">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>220</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnIngresar">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>260</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 87 9pt &quot;Arial&quot;;
background-color: rgb(0, 68, 100);
color:white;

border-radius: 10px;
}


QPushButton:hover {
background-color:rgb(0, 101, 148);
}</string>
       </property>
       <property name="text">
        <string>Ingresar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../Downloads/image-removebg-preview (4).png</normaloff>../../../../../Downloads/image-removebg-preview (4).png</iconset>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
